src/Algebra/Bundles/Raw.agda

```diff
@@ -0,0 +1,289 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Definitions of 'raw' bundles
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Algebra.Bundles.Raw where
+
+open import Algebra.Core
+open import Algebra.Structures
```

Comment:
We shouldn't need `Algebra.Structures`? And can probably get away with `Relation.Binary.Core` as well?